src/routing_node.rs

```diff
@@ -194,6 +194,14 @@ impl RoutingNode {
                     self.handle_lost_connection(connection);
                 }
                 Ok(::crust::Event::BootstrapFinished) => {
+                    match self.core.state() {
+                        &::routing_core::State::Disconnected => {
+                            self.reset();
```

Comment:
Is this event not supposed to indicate that crust has successfully bootstrapped.